src/eos-autoupdater.c

```diff
@@ -30,13 +50,17 @@ typedef enum _UpdateStep {
   UPDATE_STEP_APPLY
 } UpdateStep;
 
+/* These must be kept in sync with #UpdateStep. */
+#define UPDATE_STEP_FIRST UPDATE_STEP_NONE
+#define UPDATE_STEP_LAST UPDATE_STEP_APPLY
+
 #define SEC_PER_DAY (3600ll * 24)
 
 /* This file is touched whenever the updater starts */
-#define UPDATE_STAMP_DIR        "/var/lib/eos-updater"
-#define UPDATE_STAMP_NAME       "eos-updater-stamp"
+static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
```

Comment:
Hrm, I think it would be better to use `sharedstatedir` here, and set it to `/var/lib` (which Debian does by default, so we don’t need to do anything). The FHS defines `/var/lib` as [variable state information](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s08.html), which is what we need here.